sample-app/src/components/SearchBar.tsx

```diff
@@ -68,29 +83,43 @@ export default function SearchBar({
         placeholder={placeholder}
         screenReaderInstructions={SCREENREADER_INSTRUCTIONS}
         screenReaderInstructionsId={screenReaderInstructionsId}
-        options={autocompleteResponse?.results.map(result => {
-          return {
-            value: result.value,
-            render: () => renderWithHighlighting(result)
-          }
-        }) ?? []}
-        optionIdPrefix='Autocomplete__option'
+        screenReaderText={screenReaderText}
         onSubmit={executeQuery}
-        updateInputValue={value => {
+        onInputChange={(value) => {
```

Comment:
nit: we usually don't put the parenthesis around the param when there is just one param